demo/DemoArticles/ArticleRef.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Model for article data in ./article.json
+ * @type {object}
```

Comment:
I don't think a type is necessary here because the type is ArticleRef. If you want, you could add `@class` but I'm not sure why that's needed since the class keyword is used.